crates/aggregator/Cargo.toml

```diff
@@ -14,6 +14,7 @@ bincode = { workspace = true }
 e3-config = { workspace = true }
 e3-data = { workspace = true }
 e3-events = { workspace = true }
+e3-evm = { workspace = true }
```

Comment:
I am not convinced we should share anything from evm with aggregator as aggregator is kind of a core module and evm is shell.

We should have a core version of the FinalizeCommittee event in events we use.